extensions/raycast-bard-ai/src/api/common.jsx

```diff
@@ -122,6 +137,7 @@ export default function ResultView(prompt, toast_title, type = "text", title, op
         !loading && (
           <ActionPanel title="Actions">
             <Action.CopyToClipboard title="Copy Results" content={pasteContent} />
+            {/* <Action title="Open in Chat" onAction={() => openInChat(response)} shortcut={{ modifiers: ["cmd", "shift"], key: "k" }} icon={Icon.Message} /> */}
```

Comment:
should this be commented?

Reply:
Chat is still a work in progress. Google changed their API, breaking my code, so I am pushing this mainly to fix that...

Reply:
Well I mean the only thing that needs to be done is bump `bard-ai` version to v1.4.2, so what do you suggest? My internet is really slow right now, so making remote git changes is very tedious...

Reply:
I'd suggest marking this PR as draft, and I can open another one that updates Bard to the version that gets the extension to work again. Then you have a bit of time to implement the chat without stressing out about the extension being broken. :)

Reply:
https://github.com/raycast/extensions/pull/7320